### PR TITLE
Query - Update responsive styles in templates

### LINKED ISSUE #1580: Overflowing featured image in query block in tablet and mobile mode
### Description

Featured images in the query block are neither contained nor scaled (like in desktop mode) in the block editor. The images overflow and overlap due to being rendered full size.

### Steps to reproduce

1. Install a fresh WordPress site with GeneratePress or Astra as the theme.
1. Install GenerateBlocks 2.0 beta 3 and GenerateBlocks Pro 2.0 beta 3.
1. Add two posts with a featured image (e.g. 1920 px in width).
1. Create a new page, insert the query block, choose ”two columns & featured image” as the layout to start with.
1. Switch to tablet view or mobile view.

#### Expected behavior

The images are neither contained nor scaled like in desktop mode. The issue only affects the block editor (not the frontend).

#### Actual behavior

The images should behave like in desktop mode and on the frontend.

#### References: 

https://github.com/user-attachments/assets/8c3ea447-ddaf-427c-9b4b-29c1fdc7eba5

### PR DESCRIPTION
**Pull request by iansvo**
Closes #1580  Adds responsive styles and image containment styles. 

Requires: https://github.com/EDGE22-Studios/styles-builder/pull/33

### CODE REVIEW

**Comment by tomusborne** [comment]
We should use this helper util to get the mobile at-rule: https://github.com/tomusborne/generateblocks/pull/1587

**Comment by iansvo** [comment]
@tomusborne I updated this to use some new atRule utilities I created that have automatic fallback built in. You'll need that styles-builder branch to see the functionality.

File: src/blocks/query/templates.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
 import { __ } from '@wordpress/i18n';
 
+import { getAtRuleValue } from '@edge22/styles-builder';
+
+const mobileAtRule = getAtRuleValue( 'smallWidth' );
+
+console.log( mobileAtRule );
```
Comment (tomusborne) [inline src/blocks/query/templates.js:7]: @iansvo Gotta remove this

Comment (iansvo) [reply src/blocks/query/templates.js]: Done.

**Pull request merged**